src/Components/WordTermDisplay.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from 'react';
+import WordTerm from './WordTerm';
+
+// airtable configuration
+const Airtable = require('airtable');
+
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey })
+  .base(airtableConfig.baseKey);
+
+function WordTermDisplay() {
```

Comment:
You can probably rename this to Dictionary Display! Another option is to just move all of the code from this file to the Dictionary page file because this is the only component that will be displayed by that file so it's redundant for now!